agentscope-core/src/main/java/io/agentscope/core/tool/Toolkit.java

```diff
@@ -257,6 +262,72 @@ private void registerAgentTool(
                 groupName != null ? groupName : "ungrouped");
     }
 
+    public void registerAgentSkill(String skillDir) {
```

Comment:
We(Java)'d better support `public void registerAgentSkill(String skillContent)` and `public void registerAgentSkill(AgentSkill agentSkill)`.

Reading files is not a best practice in Java